diff --git a/build.rs b/build.rs
index 8c7bff5..2a8c385 100644
--- a/build.rs
+++ b/build.rs
@@ -10,11 +10,13 @@ fn main() {
     let out_dir_str = var("OUT_DIR").unwrap();
     let out_dir = Path::new(&out_dir_str);
 
-    if var("CARGO_FEATURE_NATIVE_LIB").ok().is_some() {
+    // Patched to always generate C code.
+    // Building using buildRustCrate does not seem to re-export the correct env variables
+    // if var("CARGO_FEATURE_NATIVE_LIB").ok().is_some() {
         // generate the C code
         generate_c_code(protocol_file, out_dir.join("wayland_c_api.rs"), Side::Client);
         generate_c_interfaces(protocol_file, out_dir.join("wayland_c_interfaces.rs"));
-    } else {
-        generate_rust_code(protocol_file, out_dir.join("wayland_rust_api.rs"), Side::Client);
-    }
+    // } else {
+    //     generate_rust_code(protocol_file, out_dir.join("wayland_rust_api.rs"), Side::Client);
+    // }
 }
